tasks/__init__.py

```diff
@@ -75,33 +75,33 @@ def test_module(ctx, module=None, params=None):
 
 
 @task
-def test_travis_on_prod(ctx):
+def test_selenium_on_prod(ctx):
     """
     Runs targeted prod smoke tests on the latest Chrome
     """
     flake(ctx)
     print('>>> Testing modules in "{}" in Chrome'.format('tests'))
-    test_travis_with_retries(
+    test_selenium_with_retries(
         ctx, 'Master', _get_test_file_list(), module=['-m', 'smoke_test']
     )
 
 
 @task
-def test_travis_part_one(ctx):
+def test_selenium_part_one(ctx):
```

Comment:
Do we still need to break the Staging tests into 2 parts? I thought that was only due to Travis limitations.